components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -899,8 +899,8 @@ this mask will have smb calculated over the entire global land surface
 <stream_year_first_popdens use_fates=".true."   sim_year="constant" sim_year_range="2000-2100" >1850</stream_year_first_popdens>
 <stream_year_last_popdens  use_fates=".true."   sim_year="constant" sim_year_range="2000-2100" >2010</stream_year_last_popdens>
 
-<stream_fldfilename_popdens hgrid="0.5x0.5"   use_cn=".true."                   >lnd/clm2/firedata/elmforc.ssp5_hdm_0.5x0.5_simyr1850-2100_c190109.nc</stream_fldfilename_popdens>
-<stream_fldfilename_popdens hgrid="0.5x0.5"   use_fates=".true."                   >lnd/clm2/firedata/elmforc.ssp5_hdm_0.5x0.5_simyr1850-2100_c190109.nc</stream_fldfilename_popdens>
+<stream_fldfilename_popdens hgrid="0.5x0.5"   use_cn=".true."    >lnd/clm2/firedata/elmforc.Li_20181205_mod_hist_SSP2_CMIP6_hdm_0.5x0.5_AVHRR_simyr1850-2100_c240906.nc</stream_fldfilename_popdens>
+<stream_fldfilename_popdens hgrid="0.5x0.5"   use_fates=".true." >lnd/clm2/firedata/elmforc.Li_20181205_mod_hist_SSP2_CMIP6_hdm_0.5x0.5_AVHRR_simyr1850-2100_c240906.nc</stream_fldfilename_popdens>
```

Comment:
@gaurabkcutk . It is also set to replace the file for configurations with FATES. Is it fine?

Reply:
@glemieux , @rgknox , The population density dataset must be updated for the big-leaf veg model. Presently, this dataset is being updated for FATES also. Is it okay to update this dataset for FATES?

Reply:
If not updating, also be aware that the file was created for 'SSP5' for BGCv1.

Reply:
Hi @bishtgautam, I think this is ok, but I'd like to check with some potential stakeholders.  I should have a decision by the end of the day today.

Reply:
@bishtgautam I'm assuming that the format of the data isn't changing between these files correct?  If that's the case, then we should be good to go.

Reply:
Thanks for checking with FATES colleagues. The data format isn't changing, but the values are changing.